src/language/mod.rs

```diff
@@ -167,6 +167,12 @@ impl Language {
 	}
 
 	/// Get the index of the word in the word list.
+	///
+	/// WARNING:
```

Comment:
```suggestion
	/// # Warning
```

Should be a H1 and maybe Titlecased and not ALL_CAPS